### PR TITLE
Create Food

### PR DESCRIPTION
**Pull request by scalderwood1**
What do you think

### CODE REVIEW

**Comment by scalderwood1** [comment]
@githubteacher

**Comment by scalderwood1** [comment]
@ersarabdeep

**Comment by ersarabdeep** [comment]
@scalderwood1 Feeling Hungry??

**Comment by scalderwood1** [comment]
Very hungry. Food in 45 min!

**Comment by scalderwood1** [comment]
What about you?

**Comment by scalderwood1** [comment]
This PR resolves #6

File: TwinLakes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Twin Lakes, WI - you guessed it - there are two lakes
+It is a small town and there is nothing but bars with terrible food and a Subway.
```
Comment (ersarabdeep) [inline TwinLakes.md:2]: Come on there's got to be some place you like to go to eat

Comment (scalderwood1) [inline TwinLakes.md:2]: Sure, outside of Twin Lakes :)

Comment (ersarabdeep) [inline TwinLakes.md:2]: :+1:

**Comment by scalderwood1** [comment]
I think its time to merge me

**Comment by ersarabdeep** [comment]
@scalderwood1 food is ready :smiley:

**Pull request merged**